activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/config/ActivitiKeycloakAutoConfiguration.java

```diff
@@ -159,13 +161,14 @@ public KeycloakClient keycloakClient(
             clientRegistrationRepository,
             clientRegistration
         );
-        KeycloakClient keycloakClient = Feign
+        return Feign
             .builder()
             .contract(new SpringMvcContract())
             .encoder(new SpringEncoder(messageConverters))
             .decoder(new SpringDecoder(messageConverters, customizers))
             .requestInterceptor(clientCredentialsAuthRequestInterceptor)
+            .logLevel(Logger.Level.FULL)
```

Comment:
can we use a property for it. Thanks

Reply:
I forgot to remove these properties on the feign